fv_be/app/models/dictionary.py

```diff
@@ -0,0 +1,154 @@
+from django.db import models
+from django.utils.translation import gettext as _
+
+# FirstVoices
+from .sites import BaseSiteContentModel, BaseControlledSiteContentModel
+from .category import Category
+from .part_of_speech import PartOfSpeech
+
+
+class Note(BaseSiteContentModel):
+    """Model for notes associated to each dictionary entry."""
+
+    # from fv:notes,fv:general_note, fv:cultural_note, fv:literal_translation, fv-word:notes, fv-phrase:notes
+    text = models.TextField()
+    dictionary_entry = models.ForeignKey(
+        "DictionaryEntry", on_delete=models.CASCADE, related_name="notes"
+    )
+
+    def __str__(self):
+        return self.text
+
+
+class Acknowledgement(BaseSiteContentModel):
+    """Model for acknowledgments associated to each dictionary entry."""
+
+    # from fv:acknowledgments, fv:source, fv:reference, fv-word:acknowledgement, fv-phrase:acknowledgement
+    text = models.TextField()
+    # todo: Confirm if this should be moved to dictionaryEntry, M:M relation
+    dictionary_entry = models.ForeignKey(
```

Comment:
I think this todo can be removed. ForeignKey looks correct to me

Reply:
@sarahfirstvoices Wouldn't it be better to make Acknowledgement a M:M relationship to avoid duplication for those teams that want to add the same acknowledgement to multiple entries?

Reply:
By pointing multiple entries to the same acknowledgement, we would introduce a behaviour where editing the acknowledgement on one entry would edit it for all other entries that have the same acknowledgement. So if you had an acknowledgement for "page 5" on ten different entries, and then you edited one to "page 6", they would all update to "page 6". I don't think this is what we want for our use case, I don't think it would be intuitive for the user. Same issue for notes.

Reply:
@caforbes If we added M:M on the acknowledgement model we wouldn't necessarily have to make use of the ability to make multiple references to all acknowledgements, but by putting this in the model we could use it for specific cases where a team wants to add the same acknowledgment to multiple entries. We already have teams who have added the same acknowledgement to whole batches, and if they ever wanted to modify or update that acknowledgment it would need to be modified on each and every one.

Reply:
Current resolution: Only foreignKey relationship for acknowledgements for now. Moreover, prefix to be added to dictionary models to be specific. If the models need to be reused, we can pull them out of DictionaryModel file and make it a separate table.